src/Loader/MiddlewareQueueLoader.php

```diff
@@ -115,7 +115,11 @@ protected function loadAuthorizationMiddleware(MiddlewareQueue $middlewareQueue,
         }
 
         $middlewareQueue->add(new AuthorizationMiddleware($plugin, Configure::read('Auth.AuthorizationMiddleware')));
-        $middlewareQueue->add(new RequestAuthorizationMiddleware());
+        
+        if (Configure::read('Auth.AuthorizationMiddleware.requireAuthorizationCheck') !== false) {
```

Comment:
I don't like this key naming. The condition is not for AuthorizationMiddleware but RequestAuthorizationMiddleware

Reply:
@steinkel ?

Reply:
Additionally you could just replace this loader with Users.middlewareQueueLoader config

Reply:
I agree, naming could be improved